include/bout/coordinates.hxx

```diff
@@ -120,6 +137,15 @@ public:
   int calcContravariant(const std::string& region = "RGN_ALL");
   int jacobian(); ///< Calculate J and Bxy
 
+  /// Return if the metrics are 3D
+  // needs to be static for old gcc
+  static constexpr bool is3D() {
+#ifdef COORDINATES_USE_3D
+    return true;
+#else
+    return false;
+#endif
```

Comment:
`use_metric_3d` or `use_3d_metric`?
Do you have a preference for the later, if so, why?

I have used the former, but would be happy to change it ...

Reply:
I have a slight preference for `use_3d_metric` because that's how I would naturally say it myself. I don't have a problem with `use_metric_3d` if that's what you're already using though.